.ci/jenkins_job_build.sh

```diff
@@ -46,6 +46,11 @@ init_ci_flags() {
 	# Use experimental kernel
 	# Values: true|false
 	export experimental_kernel="false"
+	# Use experimental qemu
+	export experimental_qemu="true"
+	# shared fs to use
+	# Values: 9pfs|virtiofs
+	export shared_fs="9pfs"
```

Comment:
@jcvenegas, my OCD is killing me.

Would be possible to have those two newly added in capital letters, just to match with the majority of the other ones we already have?
Then, in a separate commit, would be possivel to also switch experimental_kernel to all caps? :-)

Apart from that, lgtm!

Reply:
Sure :) make sense